testgrid/config/config.yaml

```diff
@@ -1039,10 +1039,12 @@ dashboards:
   - name: gce-ha-master
     test_group_name: kubernetes-e2e-gce-ha-master
 
-- name: google-gci
+# "google-gci-dev" is a private tab that should be looked at by GCI/COS
```

Comment:
I suggest rewording this as this is not 'private' by any means (permissions or anything :) ).
May be also add that when making changes here, please cc @kubernetes/goog-image .